Clair: handle messages from Clair v4

Allda opened the pull request:
In Clair v4 a message format changed a bit. NVRA is no longer available
and it was replaced by image_id.

This commit also unifies strings in Clair related files. All strings
uses double quotes now.

mikebonnet commented on fedmsg_meta_umb/clair.py:
```diff
@@ -20,82 +20,92 @@
+                template = self._(
+                    "New image scanned: {nvr}.{arch} ({grades[0][grade]})"
+                )
+            elif inner_msg.get("image_id"):
```
In what cases is `nvra` present? Why not check and display that, before defaulting to the `image_id`?

mikebonnet commented on fedmsg_meta_umb/clair.py:
```diff
@@ -20,82 +20,92 @@
-                return set([inner_msg['brew_build'].rsplit('-', 2)[0]])
+        inner_msg = msg["msg"]
+        if msg["topic"].endswith("clair.scan") and inner_msg.get("nvr"):
+            return set([inner_msg["nvr"].rsplit("-", 2)[0]])
```
You could parse a package name out of `nvra` in the same way.

mikebonnet reviewed (changes requested):
The new message format which contains a `nvra` (NVR + arch) could be better supported.

Allda commented:
NVRA is only available in Clair v4 which was deployed only to dev environment. Stage and prod are going to be released in the next weeks. Should I wait for all environment and then remove Clair v3 related code and replace it with the only v4? The current PR supports both versions.

mikebonnet commented:
> NVRA is only available in Clair v4 which was deployed only to dev environment. Stage and prod are going to be released in the next weeks. Should I wait for all environment and then remove Clair v3 related code and replace it with the only v4? The current PR supports both versions.

Supporting both formats is good (messages in the old format will still be available in the database), and I don't think there's any need to wait for deployments. My comments just meant that anywhere you're using `{nvr}.{arch}` with v3 messages, you could be using `{nvra}` with v4 messages, which would be more informative then displaying an `image_id`.

mikebonnet commented:
Could you rebase this?

Allda commented:
@mikebonnet sorry that it took me so long but I kind of forget about this PR. I added changes based on your comments, added a new message type, and rebased the PR.
Travis so very slow today so hopefully checks will pass soon.

mikebonnet commented:
This looks great, thanks!
